front/public/swagger.json

```diff
@@ -3133,17 +3148,12 @@
                 "properties": {
                   "name": {
                     "type": "string",
-                    "description": "Name of the table",
-                    "deprecated": true
```

Comment:
@overmode I believe you updated the swagger.json file directly so we are losing your changes.
You need to update in the handler files directly.

Reply:
I removed this deprecation warning for tables on purpose, but probably didn't run npm run docs, which is why they pop up now, thanks for letting me know 🙌

Reply:
Ah ok, cool then !